blockchain/block/blockstore.go

```diff
@@ -42,7 +42,6 @@ func (in *Store) ToProto() *iotextypes.BlockStore {
 
 // FromProto converts from proto message
 func (in *Store) FromProto(pb *iotextypes.BlockStore) error {
```

Comment:
are this function and the deserialize function still in use? if not, we can delete them

Reply:
done